### ISSUE TITLE
[CAL-1892] Update the layout image previews to dark mode if theme is selected (nice to have)

### ISSUE THREAD

**Issue by Jaibles** [calcom/cal.com#9381]
![CleanShot 2023-06-07 at 08.35.46@2x.png](https://uploads.linear.app/e86bf957-d82f-465e-b205-135559f4b623/4b50aad6-6886-419e-840e-211142d8d7fd/ce25c15e-12d5-46ed-9f97-da28503ef77f)

* [ ] Blocked by @Jaibles creating dark mode assets first

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-1892](https://linear.app/calcom/issue/CAL-1892/update-the-layout-image-previews-to-dark-mode-if-theme-is-selected)</sub>
